Solution1/ConsoleApp/Constants.Errors.cs

```diff
@@ -7,6 +7,8 @@ public static partial class Errors
             public const string BadCityName = "Entered incorrect city name. Try one time yet.";
             public const string RequestError = "Request error. Try again later.";
             public const string UnexpectedError = "Unexpected error. Try one time yet.";
+            public const string IncorrectValue = "Incorrect value. Try one time yet.";
```

Comment:
remove unused

Reply:
I use this

Reply:
Why it's in errors and not in validation